index.html

```diff
@@ -22,197 +22,223 @@
+              </nav>              
+            </div>
+          </div>          
         </section>
       </header>
 
       <!-- Header -->
-      <section class="header-container">      
-        <h1 class="about-me"><b>I'm Inoulen Glad to see you!</b></h1>            
-        <p class="header-main-text">I'm a software developer! I can help you build a product, 
-          feature or website. Look through through some of my work and experience! If you like 
-          what you see and have a project you need coded, don't hesitate to contact me.
-        </p>        
-        <h6 class="connect">LET’S CONNECT</h6>
+      <section class="header-container"> 
+        <div class="header-main">
+          <h1 class="about-me">I'm Inoulen </br>Glad to see you!</h1>
+          <p class="header-main-text">I'm a software developer! I can help you build a product, 
+            feature or website. Look through through some of my work and experience! If you like 
+            what you see and have a project you need coded, don't hesitate to contact me.
+          </p>        
+          <h6 class="connect">LET’S CONNECT</h6>
 
-        <!-- Social media icons section -->
-        <ul class="iconlists">
-          <li>
-            <a href="https://twitter.com/"><img src="./images/Twitter.png" alt="" /></a>
-          </li>
-          <li>
-            <a href="https://linkedin.com/"><img src="./images/Linkedin icon.png" alt="" /></a>
-          </li>
-          <li>
-            <a href="https://medium.com/"><img src="./images/Vector 2.png" alt="" /></a>
-          </li>
-          <li>
-            <a href="https://github.com/"><img src="./images/Vector 1.png" alt="" /></a>
-          </li>
-          <li>
-            <a href="https://shutterstock.com/"><img src="./images/Vector 3.png" alt="" /></a>
-          </li>          
-        </ul>      
+          <!-- Social media icons section -->
+          <ul class="iconlists">
+            <li>
+              <a href="https://twitter.com/"><img src="./images/Twitter.png" alt="" /></a>
+            </li>
+            <li>
+              <a href="https://linkedin.com/"><img src="./images/Linkedin icon.png" alt="" /></a>
+            </li>
+            <li>
+              <a href="https://medium.com/"><img src="./images/Vector 2.png" alt="" /></a>
+            </li>
+            <li>
+              <a href="https://github.com/"><img src="./images/Vector 1.png" alt="" /></a>
+            </li>
+            <li>
+              <a href="https://shutterstock.com/"><img src="./images/Vector 3.png" alt="" /></a>
+            </li>          
+          </ul>      
+        </div>
       </section>
 
       <!-- Middle section - Works -->
       <section class="card-works-wrapper">
 
         <!-- Card 1 -->
-        <div class="card-works-container card-size1">
+        <div id="order-1" class="card-works-container">          
           <img src="./images/Snapshoot-Portfolio.png" alt="Project1">
```

Comment:
- [ ] We have a small issue to fix on this card section, seeing the image below, the space on the left is small than the one on the right, which makes your cards not centered properly, would you mind if you fix that up? not only for this one card but all of them.

<img width="402" alt="2" src="https://user-images.githubusercontent.com/77843050/208069465-8f52dad6-176b-4a87-af02-7b8a056020b8.png">

Reply:
Hi @joskalenda, thanks for the detailed feedback. I have fixed the alignment to center the cards in the middle with equal gaps.